### PR TITLE
Send all locations in JET stacktraces

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by Klafyvel** [comment]
Hi! It seems Runic.jl was updated since the last time I ran it on the code. Can you run it once on the whole repository to fix the CI? Thanks!

**Comment by fredrikekre** [comment]
I think that looks correct, it is indented as a continuation of the previous line. However, perhaps that expression would look better as a single line one:
```julia
frames = [(frame.file, frame.line, frame.func) for frame in stackframes]
```

**Comment by Klafyvel** [comment]
> I think that looks correct, it is indented as a continuation of the previous line. However, perhaps that expression would look better as a single line one:
> 
> ```julia
> frames = [(frame.file, frame.line, frame.func) for frame in stackframes]
> ```

Ok, I see the point. In that case I prefer the one-line indeed.

File: .github/workflows/Runic.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: julia-actions/cache@v2
       - name: Install Runic
         run: |
-          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl")'
+          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl", rev = "3931a4ffe3e46220e72829de367a9366de1684d2")'
       - name: Run Runic
         run: |
-          julia --color=yes -m Runic --check --diff $(git ls-files -- '*.jl')
+          git ls-files -z -- '*.jl' | xargs -0 julia --project=@runic -m Runic --check --diff
```
Comment (fredrikekre) [inline .github/workflows/Runic.yml:25]: I updated the README in Runic to this which should be safer against e.g. spaces in file names etc.

File: .github/workflows/Runic.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: julia-actions/cache@v2
       - name: Install Runic
         run: |
-          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl")'
+          julia --color=yes -e 'using Pkg; Pkg.add(url = "https://github.com/fredrikekre/Runic.jl", rev = "3931a4ffe3e46220e72829de367a9366de1684d2")'
```
Comment (fredrikekre) [inline .github/workflows/Runic.yml:22]: Lock to a specific commit for now since I am still making changes to Runic.

**Comment by Klafyvel** [comment]
Looking good, thanks!

**Pull request merged**